### PR TITLE
Replace admin

### LINKED ISSUE #55: s/Admin/STSPV
Maybe not in the code for this one, but it'd be good to be clear about who 'admins' are.

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #55

### CODE REVIEW

File: app/views/users/show.html.haml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   %b Active:
   = @user.active
 %p
-  %b Admin:
+  %b STSPV:
   = @user.admin
```
Comment (dfaulken) [inline app/views/users/show.html.haml]: While we're here, can we change this to not show `true` or `false` in a view?

Comment (mboneil10) [reply app/views/users/show.html.haml]: What do you mean?

Comment (mboneil10) [reply app/views/users/show.html.haml]: I can't find it how do I rails/ruby/haml

Comment (dfaulken) [reply app/views/users/show.html.haml]: I mean that `@user.admin` is the value `true` or `false`. Ideally this would say "Yes" or "No".

**Pull request merged**